pkg/devenvironment/name.go

```diff
@@ -0,0 +1,57 @@
+package devenvironment
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/okteto/okteto/pkg/k8s/configmaps"
+	oktetoLog "github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/model"
+	"github.com/okteto/okteto/pkg/okteto"
+	"k8s.io/client-go/kubernetes"
+)
+
+// InferName infers the application name from the folder received as parameter
```

Comment:
```suggestion
// InferName infers the dev environment name from the folder received as parameter
```

should this be regarding dev environment although it is an application?